### PR TITLE
Funqy Google Cloud Functions codestart

### PR DESCRIPTION
**Pull request by ia3andy**
Supersede https://github.com/quarkusio/quarkus/pull/14279

### CODE REVIEW

File: devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/singleton-examples/funqy-google-cloud-functions-example/codestart.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: funqy-google-cloud-functions-example
+ref: funqy-google-cloud-functions
+type: code
+tags:
+  - example
+  - singleton-example
+metadata:
+  title: Funqy Google Cloud Functions Integration examples
+  description: TExamples of Background Functions with Funqy for Quarkus.
```
Comment (loicmathieu) [inline devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/singleton-examples/funqy-google-cloud-functions-example/codestart.yml:9]:
```suggestion
  description: Examples of Background Functions with Funqy for Quarkus.
```

**Review by loicmathieu — APPROVED** [review]
LGTM

**Comment by ia3andy** [comment]
I added a small fix to this PR to allow this:
```
mvn io.quarkus:quarkus-maven-plugin:999-SNAPSHOT:create -Dexamples=funqy-google-cloud-functions 
```

**Comment by ia3andy** [comment]
Ok @loicmathieu  I've discussed with @gsmet, he needs all the CI for some top priority PRs, this one will not make the CR1 but no worries it will be backported in the next CR of Final 1.12

**Comment by gsmet** [comment]
Restarted CI.

**Pull request merged**